From 8261df38129ffb38583349f30169500b7583b138 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Sat, 24 Mar 2018 13:29:00 +0800
Subject: [PATCH] pinctrl: rockchip: Add rtc 32k io route data for rk3308

If rtc_32k pin iomux is enabled, the grf_rtc_32k_ioe
needs to be enabled also.

Change-Id: I24d04c09b1190aa0cd323231c2f2e27ae900d200
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 drivers/pinctrl/pinctrl-rockchip.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/pinctrl/pinctrl-rockchip.c b/drivers/pinctrl/pinctrl-rockchip.c
index a5bc6a101a2c..1cfd9d424a68 100644
--- a/drivers/pinctrl/pinctrl-rockchip.c
+++ b/drivers/pinctrl/pinctrl-rockchip.c
@@ -1006,6 +1006,13 @@ static struct rockchip_mux_route_data rk3228_mux_route_data[] = {
 
 static struct rockchip_mux_route_data rk3308_mux_route_data[] = {
 	{
+		/* rtc_clk */
+		.bank_num = 0,
+		.pin = 19,
+		.func = 1,
+		.route_offset = 0x314,
+		.route_val = BIT(16 + 0) | BIT(0),
+	}, {
 		/* uart2_rxm0 */
 		.bank_num = 1,
 		.pin = 22,
-- 
2.35.3

